explainer.md

```diff
@@ -335,13 +403,68 @@ not yet resolved. This causes work that is still needed to become synchronous,
 enabling the idle-until-urgent pattern.
 
 
-##### DisplayLockContext.updateAndCommit()
+#### DisplayLockContext.updateAndCommit()
 
 This operation combines the effects of an `update()` and a `commit()` calls:
 * It causes the element to be co-operatively updated.
 * When the update is finished, the element is committed resulting in visual
   updates to appear on screen.
 
+
+#### Element activation
+
+When an element is locked,
+there are some actions in the page that might require the element to get unlocked and be rendered to work properly.
+
+If the browser needs to *activate* a locked element
+or an element that has one or more locked ancestors,
+in order to make the activated element be visible,
+it will commit all locked elements in the ancestor chain of the element to be activated,
+unless at least one of them is locked without the `activatable` flag set to `true`.
+
+*Activating an element* is defined as one of the following actions:
+ - `focus()` is called on the element
+ - `scrollIntoView()` is called on the element
+ - Tab order navigation lands on the element
+ - Anchor link navigation navigates to the element
+ - Find-in-page active match navigation goes to the element
+
+For all the elements that we commit as part of element activation,
+we will send a `beforeactivate` event to it just before committing.
+
+#### beforeactivate event
+
+| property  | value  |
+|---|---|
+| bubbles  | true  |
+| composed | false |
+| target  | the previously-locked ancestor |
```

Comment:
Nearest previously-locked ancestor?

Reply:
Since there can be multiple ancestor, not just nearest, I used "previously-locked ancestor" instead (and removed "the"). Maybe there's a better way to phrase this?